### PR TITLE
Update schema-utils

### PR DESCRIPTION
**Pull request by snitin315**
**What kind of change does this PR introduce?**
chore, update `schema-utils`

Fix https://github.com/webpack/webpack/pull/13713

**Did you add tests for your changes?**
Yes updated snaps

**Does this PR introduce a breaking change?**
No

**What needs to be documented once your changes are merged?**
Nothing

### CODE REVIEW

File: types.d.ts [hunk 1]
```diff
@@ -80,7 +80,8 @@ import {
 	WithStatement,
 	YieldExpression
 } from "estree";
-import { ValidationError, validate as validateFunction } from "schema-utils";
```
Comment (sokra) [inline types.d.ts:83]: Is `ValidationError` no longer exposed from the package directly?

Comment (snitin315) [reply types.d.ts:83]: It is exposed directly https://github.com/webpack/schema-utils/blob/3915bae9a2ce3206dff5f7f3dff717af2ae1d527/src/index.js#L3 ,
Not sure why this changed.

Comment (sokra) [reply types.d.ts:83]: Anyway. I'll accept that for now. That seem to be a problem in the typings generation, which may need to be fixed

**Comment by sokra** [comment]
Thanks

**Pull request merged**